src/App.js

```diff
@@ -192,8 +199,7 @@ function renderAction(
   };
   if (
     action.animation_name === 'simple_attack' &&
-    attackTurn >= -1 &&
-    attackTurn <= 0
+    attackTurn === 0
```

Comment:
Ah, that was meant to be a long attack animation. not two short ones. :( Thanks for finding this, explains a lot! Sending a pr to restore the original intention without the bug. I think it's better to see the bullets a bit longer.